src/confluence/read/base.py

```diff
@@ -7,7 +8,42 @@ class BaseReader(object):
     This object acts as an iterator, serving data from the source
     one chunk at a time, for example, line-by-line if the derived
     reader is to a local file object.
+
+    Example
+    =======
+    :code:`python`
+        reader = BaseReader()
+        reader.open('foo.txt')
+        for line in reader:
+            print(line)
+        reader.close()
+
+    :code:`python`
+        reader = BaseReader('foo.txt')
+        for line in reader:
+            print(line)
+        reader.close()
     """
+
```

Comment:
Remove these and replace with abstract API methods __iter__, __next__, and read.